src/config/swagger.config.ts

```
@@ -0,0 +1,15 @@
+export function setupSwagger(app: INestApplication) {
+  const config = new DocumentBuilder()
+    .setTitle('Revo Backend API')
+    .setDescription('API documentation for Revo Backend')
+    .setVersion('1.0')
+    .build();
+
+  const document = SwaggerModule.createDocument(app, config);
+  SwaggerModule.setup('api/docs', app, document, {
+    jsonDocumentUrl: 'docs-json',
+  });
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)